libpod/runtime_ctr.go

```diff
@@ -159,6 +164,10 @@ func (r *Runtime) LookupContainer(idOrName string) (*Container, error) {
 		return nil, ErrRuntimeStopped
 	}
 
+	return r.lookUpContainer(idOrName)
+}
+
+func (r *Runtime) lookUpContainer(idOrName string) (*Container, error) {
```

Comment:
Why do we need this extra function?

Reply:
I was worried about the lock hanging again, but not needed anymore. fixed